doc/installation.md

```diff
@@ -8,31 +8,21 @@ issue is welcome. :)
 ## Preparation
 
 1. Install [Docker Community Edition](https://docs.docker.com/install/) on your Macbook.
-1. Build and run a dockerized MySQL server (In this example, called sqlflowdata) following [example/datasets](/example/datasets). Note that there is no need to install a local MySQL server, in which case you will have a port conflict in 3306. 
-1. Pull the latest SQLFlow Docker image: 
+1. Pull the latest SQLFlow "all-in-one" Docker image, which contains pre-built SQLFlow
+   binary, sample datasets (under `example/datasets`), and jupyter notebook.
 
    ```
-   docker pull sqlflow/sqlflow:latest
+   docker pull sqlflow/quickstart:latest
```

Comment:
Shall we release `sqlflow/quickstart:latest` to DockerHub after every successful build? This can be done at `scripts/deploy.sh`.

Reply:
Sure, will add!